### PR TITLE
Code release for TIARA

### PR DESCRIPTION
**Pull request by ArleneYuZhiwei**
Here are code files for TIARA.

### CODE REVIEW

File: papers/TIARA/TIARA/README.md [hunk 1]
```diff
@@ -0,0 +1,284 @@
+# TIARA
+
```
Comment (tellarin) [inline papers/TIARA/TIARA/README.md:2]: We don't need this 2nd README, right?

File: papers/TIARA/azure-pipelines.yml [hunk 2]
```diff
@@ -0,0 +1,69 @@
+# Python package
```
Comment (tellarin) [inline papers/TIARA/azure-pipelines.yml:1]: This file should not be part of the release.

File: papers/TIARA/TIARA/src/retriever/freebase_cache_utils.py [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+            if count % 100 == 0:
+                print('[INFO] ' + file_path + ' rdf type: ' + str(count) + ' / ' + str(len(uri_set)))
+            count += 1
+
+    pickle_save(res, freebase_cache_dir + '/rdf_type.bin')
+
+
+if __name__ == "__main__":
+    # create_relation_cache()
+    # create_neighbor_cache()
```
Comment (tellarin) [inline papers/TIARA/TIARA/src/retriever/freebase_cache_utils.py:149]: We'll provide all pre-gen caches in the data download, right?

**Review by tellarin — CHANGES_REQUESTED** [review]
Partial review one. Looks mostly good so far.

File: papers/TIARA/azure-pipelines.yml (hunk 2 above)
Comment (ArleneYuZhiwei) [reply papers/TIARA/azure-pipelines.yml:1]: I will remove this file.

File: papers/TIARA/TIARA/src/retriever/freebase_cache_utils.py (hunk 3 above)
Comment (ArleneYuZhiwei) [reply papers/TIARA/TIARA/src/retriever/freebase_cache_utils.py:149]: All the caches can be generated by our scripts

**Review by tellarin — APPROVED** [review]

**Pull request merged**